portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,33 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Noe's Portfolio</h1>
+
+      <div class = "menu">
+        <a class = "active" href="index.html">About Me</a>
+        <a href="projects.html">My Projects</a>
+        <a href="talents.html">Hidden Talents</a>
+      </div>
+
+      <p>Hi! My name is Noe, and I am a rising junior computer science major at UCI. I am from San Diego, and I enjoy playing soccer, playing video games, and watching Youtube in my free time.</p>
+      <img class = "helmet" src = "/images/IMG_2360.JPG" alt = "Me in a helmet">
+
+      <p>I went to Barcelona last summer to visit family, and while I was there, I was lucky enough to watch my favorite soccer team play. I took this picture before the game.</p>
+      <img class = "stadium" src = "/images/Image-1.jpg" alt = "Barca picture" >
+
+      <p>During that same summer, I got to see Portugal and Holland for the first time, which was really cool! I took the picture to the left in Porto, and the picture from the right is from Utrecht.
+      <div class = "row">
+          <div class = "column">
```

Comment:
The Google style guide suggests to use 2 space indention: https://google.github.io/styleguide/htmlcssguide.html#Indentation

Please fix here and other places.